src/components/Header/Header.jsx

```diff
@@ -0,0 +1,255 @@
+import React, { useState } from "react"
+import { useLocation } from 'react-router-dom'
+import { NavLink } from "react-router-dom"
```

Comment:
You could use one line to import all what you want from react router

Reply:
Oops, really. I'll fix it)